### PR TITLE
Add more trace logs to found out issue with UI data

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: util/SerializeMultiplayerCommon.cpp [hunk 1]
```diff
@@ -84,13 +87,16 @@ void SaveGameUIData::serialize(Archive& ar, const unsigned int version)
                     ids_obsolete[id_pair_pair.first] = id_pair_pair.second.get();
             }
             // serialize into archive
+            TraceLogger() << "SaveGameUIData::serialize desing data into archive";
             ar  & BOOST_SERIALIZATION_NVP(ordered_ship_design_ids)
                 & BOOST_SERIALIZATION_NVP(ids_obsolete);
-
+            TraceLogger() << "SaveGameUIData::serialize desing data into archive completed";
```
Comment (geoffthemedio) [inline util/SerializeMultiplayerCommon.cpp]: desing -> design

File: util/SerializeMultiplayerCommon.cpp [hunk 2]
```diff
@@ -63,11 +63,14 @@ template void SinglePlayerSetupData::serialize<freeorion_xml_iarchive>(freeorion
...
 void SaveGameUIData::serialize(Archive& ar, const unsigned int version)
 {
+    TraceLogger() << "SaveGameUIData::serialize " << (Archive::is_saving::value ? "saving" : "loading")
+                  << " version " << version;
     ar  & BOOST_SERIALIZATION_NVP(map_top)
         & BOOST_SERIALIZATION_NVP(map_left)
         & BOOST_SERIALIZATION_NVP(map_zoom_steps_in)
         & BOOST_SERIALIZATION_NVP(fleets_exploring)
         & BOOST_SERIALIZATION_NVP(obsolete_ui_event_count);
+    TraceLogger() << "SaveGameUIData::serialize processed obsolete UI event count ";
```
Comment (geoffthemedio) [inline util/SerializeMultiplayerCommon.cpp]: remove extra space

File: util/SerializeMultiplayerCommon.cpp (hunk 1 above)
Comment (o01eg) [reply util/SerializeMultiplayerCommon.cpp]: Fixed.

File: util/SerializeMultiplayerCommon.cpp (hunk 2 above)
Comment (o01eg) [reply util/SerializeMultiplayerCommon.cpp]: Fixed.

**Pull request merged**